lib/cloudapi2.js

```diff
@@ -2318,17 +2318,27 @@ CloudApi.prototype.createVolume = function createVolume(options, cb) {
     assert.object(options, 'options');
     assert.optionalString(options.name, 'options.name');
     assert.optionalNumber(options.size, 'options.size');
-    assert.optionalArrayOfUuid(options.networks, 'options.networks');
```

Comment:
We could also just push `net.uuid` at https://github.com/joyent/node-triton/blob/tritonnfs/lib/do_volume/do_create.js#L65 instead of pushing the network object. I don't have a preference, but it might require less changes.